File: cfgs/pythoncfgs/SORT_EVEN_NUMBERS_ASCENDING_ORDER_SORT_ODD_NUMBERS_DESCENDING_ORDER_1.py
Nodes
1;conditional;foo < foo;8;
2;statement;fun_1();11;
3;statement;foo < foo;12;
4;conditional;foo_1[foo_2] & 1;13;
5;statement;foo_1[foo_2] *= -1;14;
6;conditional;foo_1[foo_2] & 1;9;
7;statement;foo_1[foo_2] *= -1;10;
8;exit;;
Edges
0;2;6;
1;3;
2;4;
3;3;5;
4;3;
5;1;7;
6;1;
7;

File: cfgs/java_cfgs_foo/SORT_EVEN_NUMBERS_ASCENDING_ORDER_SORT_ODD_NUMBERS_DESCENDING_ORDER_1.java
Nodes
1;special;;;
2;statement;int i = 0;14;
3;conditional;foo_1 < foo_2;15;
4;conditional;(foo_1[foo_2] & 1) != 0;16;
5;statement;foo_1[foo_2] *= -1;16;
6;statement;i++;16;
7;statement;fun_1(foo_1);17;
8;statement;int i = 0;18;
9;conditional;foo_1 < foo_2;19;
10;conditional;(foo_1[foo_2] & 1) != 0;20;
11;statement;foo_1[foo_2] *= -1;20;
12;statement;i++;20;
13;exit;;;
Edges
0;2;
1;3;
2;4;7;
3;5;6;
4;6;
5;3;
6;8;
7;9;
8;10;13;
9;11;12;
10;12;
11;9;
12;
